core/federated/federate.c

```diff
@@ -1875,6 +1669,17 @@ void handle_message(int socket, int fed_id) {
     _lf_schedule_value(action, 0, message_contents, length);
 }
 
+void stall_advance_level_federation(environment_t* env, size_t next_reaction_level) {
```

Comment:
Comments can be found in [include/core/federated/federate.h](https://github.com/lf-lang/reactor-c/pull/191/files#diff-cb2a82eec514eef994f20ceedc7c9238714cfe86035befb4880eac6913316912). I think we have agreed that it is better to keep comments for global functions in header files instead of duplicating them.